ember-cli-build.js

```diff
@@ -5,6 +5,9 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 module.exports = function(defaults) {
   const app = new EmberApp(defaults, {
     // Add options here
+    babel: {
+      includePolyfill: true
+    }
```

Comment:
Which polyfill was needed for this?
Also this throws a deprecation. The correct syntax today is:
```
  'ember-cli-babel': {
    includePolyfill: true
  }
```